airbyte-cdk/python/airbyte_cdk/sources/file_based/config/csv_format.py

```diff
@@ -64,6 +64,11 @@ class Config:
         default=[],
         description="A set of case-sensitive strings that should be interpreted as null values. For example, if the value 'NA' should be interpreted as null, enter 'NA' in this field.",
     )
+    strings_can_be_null: bool = Field(
```

Comment:
Should we have some validation on this that requires `null_values` to be set if `strings_can_be_none` is True?

Reply:
I don't think we want that validation. Users should be able to configure `null_values` even if `strings_can_be_none` is False to set non-string fields to `None`

Reply:
Oh right, makes sense!